brd-ios/breadwallet/src/Redesign/Views/MainSwapView.swift

```diff
@@ -0,0 +1,201 @@
+// 
+//  MainSwapView.swift
+//  breadwallet
+//
+//  Created by Kenan Mamedoff on 05/07/2022.
+//  Copyright © 2022 Fabriik Exchange, LLC. All rights reserved.
+//
+//  See the LICENSE file at the project root for license information.
+//
+
+import UIKit
+import SnapKit
+
+struct MainSwapConfiguration: Configurable {
+    var shadow: ShadowConfiguration?
```

Comment:
a configuration normally gives u the chance to modify all view elements... 
(like all labels / imgs / ...) 
so these needs to be expanded a bit :)